backend/tests/helpers/constants.py

```diff
@@ -70,4 +70,4 @@
 LOW_UQ_SCORE = Decimal("0.01")
 
 UQ_THRESHOLD_NOT_MAINNET = 5
-UQ_THRESHOLD_MAINNET = 20
+UQ_THRESHOLD_MAINNET = 15
```

Comment:
Isn't it weird that you keep constants doubled here and in `app`? I'm just passing by, but it caught my attention.

Reply:
Good q, but one is for the sake of testing, another for the functionality. Thanks to keeping them separately we may impact on the way how testing is coducted without having any effect on the logic part of our aplication (keeping them togehter may be considered as an acceptable way as well though)